tests/unit/amo/components/TestInstallButtonWrapper.js

```diff
@@ -34,46 +66,68 @@ import {
   fakeInstalledAddon,
   fakeTheme,
   fakeVersion,
+  getFakeAddonManagerWrapperWithJest as getFakeAddonManagerWrapper,
+  getFakeLoggerWithJest as getFakeLogger,
   render as defaultRender,
   screen,
   userAgents,
   userAgentsByPlatform,
 } from 'tests/unit/helpers';
 
 jest.mock('amo/tracking', () => ({
+  ...jest.requireActual('amo/tracking'),
   sendEvent: jest.fn(),
 }));
 
+const INVALID_TYPE = 'not-a-real-type';
+
 describe(__filename, () => {
+  let addon;
   let store;
+  let _addonManager;
+  let _getClientCompatibility;
 
   beforeEach(() => {
+    _getClientCompatibility = jest.fn().mockReturnValue({ compatible: true });
+    addon = {
```

Comment:
This can just be `addon = fakeAddon`;

Reply:
I've kept the de-structuring to ensure we got a new object every time, but made it a one-liner.